src/components/MobileApp/ModelFilterForm.js

```diff
@@ -0,0 +1,254 @@
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import cx from 'classnames/dedupe'
+import f from 'lodash'
+
+// import { action } from '@storybook/addon-actions'
+// import { Let } from '../Util'
+// import Icon from '../Icons'
+// import { ImgPlaceholder } from './ImageThumbnail'
+
+const DEFAULT_TERM = ''
+const DEFAULT_POOL_ID = 'all'
+const DEFAULT_AVAILABLE_BETWEEN = false
+const DEFAULT_START_DATE = ''
+const DEFAULT_END_DATE = ''
+const DEFAULT_QUANTITY = 1
+
+// const NOOP = () => {}
+
+const ModelFilterForm = ({
+  className = '',
+  user = {},
+  delegations = [],
+  initialTerm = DEFAULT_TERM,
+  initialUserId = user.id,
+  initialPoolId = DEFAULT_POOL_ID,
+  initialAvailableBetween = DEFAULT_AVAILABLE_BETWEEN,
+  initialStartDate = DEFAULT_START_DATE,
+  initialEndDate = DEFAULT_END_DATE,
+  initialQuantity = DEFAULT_QUANTITY,
+  pools,
+  onClear,
+  onSubmit
+}) => {
+  const [term, setTerm] = useState(initialTerm)
+  const [userId, setUserId] = useState(initialUserId)
+  const [poolId, setPoolId] = useState(initialPoolId)
+  const [availableBetween, setAvailableBetween] = useState(initialAvailableBetween)
+  const [startDate, setStartDate] = useState(initialStartDate)
+  const [endDate, setEndDate] = useState(initialEndDate)
+  const [quantity, setQuantity] = useState(initialQuantity)
+
+  const clear = () => {
+    setTerm(DEFAULT_TERM)
+    setUserId(user.id)
+    setPoolId(DEFAULT_POOL_ID)
+    setAvailableBetween(DEFAULT_AVAILABLE_BETWEEN)
+    setStartDate(DEFAULT_START_DATE)
+    setEndDate(DEFAULT_END_DATE)
+    setQuantity(DEFAULT_QUANTITY)
+  }
+
+  const queryArgs = () => {
+    return f
+      .chain({
```

Comment:
we dont want to use `.chain()` (for technical reasons https://medium.com/bootstart/why-using-chain-is-a-mistake-9bc1f80d51ba)